### PR TITLE
OV-135: Convert-fa-icons

### PR DESCRIPTION
**Pull request by JKaypa**
Convert fa icons to Chakra Icons type, remove unnecessary as={FontAweseomeicon} property in all codebase

### CODE REVIEW

**Comment by o-nedashkivska** [comment]
Please, fix conflicts

File: frontend/src/bundles/studio/components/video-menu/components/mock/menu-mock.tsx [hunk 1]
```diff
@@ -1,8 +1,27 @@
...
 const AvatarsContent: React.FC = () => <div>This is the Avatars content.</div>;
+const ScriptHeader: React.FC = () => (
+    <Flex justifyContent={'space-between'} w={'280px'}>
+        <div>Script</div>
+        <div>
+            <Icon as={IconName.UPLOAD} />
+        </div>
+    </Flex>
+);
+const ScriptContent: React.FC = () => <div>This is the Script content.</div>;
```
Comment (o-nedashkivska) [inline frontend/src/bundles/studio/components/video-menu/components/mock/menu-mock.tsx:16]: We don't need this mocked data for scripts, it is not used in the project anymore

Comment (o-nedashkivska) [reply frontend/src/bundles/studio/components/video-menu/components/mock/menu-mock.tsx:16]: Please delete it

File: frontend/src/bundles/common/icons/helper/icon-conversion.helper.ts
Comment (nikita-remeslov) [inline frontend/src/bundles/common/icons/helper/icon-conversion.helper.ts:1]: looks like we can avoid using `Icon` postfix

**Review by nikita-remeslov — CHANGES_REQUESTED** [review]

**Review by o-nedashkivska — APPROVED** [review]

**Review by anton-otroshchenko — APPROVED** [review]

**Review by o-nedashkivska — DISMISSED** [review]

**Review by o-nedashkivska — APPROVED** [review]

**Pull request merged**